api/v1alpha1/ndb_api.go

```diff
@@ -80,23 +80,25 @@ func GetDatabaseById(ctx context.Context, ndbClient *ndbclient.NDBClient, id str
 		log.Error(err, "no database id provided")
 		return
 	}
-	res, err := ndbClient.Get("databases/" + id)
+
+	getDbDetailedPath := fmt.Sprintf("databases/%s?detailed=true", id)
```

Comment:
prefer to have a separate file named "url_constants" where all NDB API endpoints are listed.